src/main/java/de/novatec/ddd/object/Aggregate.java

```diff
@@ -1,4 +1,4 @@
-package de.novatec.domainprimitives.object;
+package de.novatec.ddd.object;
```

Comment:
The artifact containing the code will still be called `domainprimitives-java`. I think we should call the base package `en.novatec.ddd.domainprimitives`.
In this example the package for the `Aggregate` class would then be in `de.novatec.ddd.domainprimitives.object`.